saleor/order/migrations/tasks/saleor3_18.py

```diff
@@ -0,0 +1,56 @@
+from django.db.models import F, Func, Min, OuterRef, Subquery
+
+from ....celeryconf import app
+from ...models import Order, OrderLine
+
+# celery task for one batch took 0.1s and 10.54MB at its peak
+BATCH_SIZE = 1000
+
+
+@app.task
+def update_order_subtotals(batch_id=None):
+    if batch_id is None:
+        # Get the minimum order number from the database
+        min_order_number = Order.objects.aggregate(Min("number"))["number__min"] or 0
```

Comment:
Can't you just make `Order.objects.last().number`?

Reply:
did you mean `Order.objects.first().number` ?